packages/issuer/lib/VcIssuer.ts

```diff
@@ -55,7 +62,7 @@ export class VcIssuer {
   /**
    * issueCredentialFromIssueRequest
    * @param issueCredentialRequest a credential issuance request
-   * @param preAuthorizedCode the pre-authorized code
+   * @param state the state to be passed to the state manager
```

Comment:
Update params to be inline with actual method params

Reply:
Fixed!